code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -695,4 +695,7 @@
 	description = "Fluorosulfuric acid is a an extremely corrosive chemical substance."
 	color = "#8E18A9" // rgb: 142, 24, 169
 	toxpwr = 2
-	acidpwr = 20
+	acidpwr = 42.0
```

Comment:
Only two are, the rest are not. However, armor values (any armor value) give the suits a chance to resist destruction.